database/mssql/scripts/dbscripts/PSP_PIMS_S24_00/Build/04_PSP_PIMS_PMBC_Tables_Build.sql

```diff
@@ -0,0 +1,294 @@
+-- Script generated by Aqua Data Studio Schema Synchronization for MS SQL Server 2016 on Tue Oct 05 16:47:03 PDT 2021
+-- Execute this script on:
+-- 		PIMS_S09_01/pmbc - This database/schema will be modified
+-- to synchronize it with MS SQL Server 2016:
+-- 		PIMS_S09_02/pmbc
+
+-- We recommend backing up the database prior to executing the script.
+
+SET XACT_ABORT ON
+GO
+SET TRANSACTION ISOLATION LEVEL SERIALIZABLE
+GO
+BEGIN TRANSACTION
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Create PMBC schema
+IF NOT EXISTS (SELECT * 
+               FROM   sys.schemas 
+               WHERE  name = N'pmbc')
+  EXEC('CREATE SCHEMA [pmbc]');
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+
+-- Drop table pmbc.PMBC_PARCEL_POLYGON_FABRIC
+PRINT N'Drop table pmbc.PMBC_PARCEL_POLYGON_FABRIC'
```

Comment:
@dfilteau can we change this to not drop the pmbc table, just only create it if it doesn't exist? This file is the reason that the pmbc table was empty in all of our environments.